db/migrate/20230512124828_add_status_checked_at_to_projects.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusCheckedAtToProjects < ActiveRecord::Migration[5.2]
+  def change
+    add_column :projects, :status_checked_at, :datetime
```

Comment:
ok I just realized if we're going to query for the oldest of these, we probably should index this.